application/src/main/data/json/edge/instructions/install/centos/instructions.md

```diff
@@ -74,12 +74,12 @@ sudo systemctl enable --now postgresql-15
 sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
```

Comment:
it is good to have an ARM repo as well. i believe thera a lot of examples how to install from platform related repo
See https://www.postgresql.org/download/linux/redhat/

*End of Life CentOS 8*

The end of support (EOS) dates for CentOS 7 and 8 are as follows: CentOS 8 - December 31, 2021.

Why we are still updating something for a dead OS?